app/isomorphic/components/header/nav-bar/index.js

```diff
@@ -1,24 +1,48 @@
-import React from "react";
+import React, { useState, lazy, Suspense } from "react";
 import get from "lodash/get";
+import { object, bool } from "prop-types";
+
 import { NavbarSearch } from "../navbar-search";
 import { MenuItem } from "../helper-components";
 import { AppLogo } from "../app-logo";
+
 import "./styles.m.css";
-function NavBar(props) {
+
+const NavBar = ({ menu, enableLogin }) => {
+  const AccountModal = lazy(() => import("../../login/AccountModal"));
+  const [showAccountModal, setShowAccountModal] = useState(false);
   return (
     <React.Fragment>
       <AppLogo />
       <ul styleName="navbar">
-        {get(props, ["menu", "default"], []).map((item, index) => {
+        {get(menu, ["default"], []).map((item, index) => {
           return (
-            <li key={`${item["id"]}${index}`} onClick={props.closeMenu} styleName="menu-item">
+            <li key={`${item.id}${index}`} styleName="menu-item desktop-view">
               <MenuItem item={item} />
             </li>
           );
         })}
+        {enableLogin && (
+          <li>
+            <button onClick={() => setShowAccountModal(true)}>Login</button>
+            {showAccountModal && (
+              <Suspense fallback={<div></div>}>
+                <AccountModal onBackdropClick={() => setShowAccountModal(false)} checkForMemberUpdated={{}} />
+              </Suspense>
+            )}
+            {/* checkForMemberUpdated from
+          withmember of qt components library and also add onBackdropClick which is used for closing popup */}
```

Comment:
remove this , if not required

Reply:
is required, it will come to use once we implement the library